tests/test_kafka_streams.py

```diff
@@ -14,20 +16,18 @@ class MyTestProcessor(BaseProcessor):
     pass
 
 
-class TestKafkaStreams(unittest.TestCase):
+def test_Given_StreamAlreadyStarted_When_CallStartAgain_Then_RaiseError():
```

Comment:
Can we stick with more Pythonic names (separated by underscores)?

Reply:
I corrected others but missed this - will fix.